### ISSUE TITLE
Offer page building on Service pages

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov#703]
Page building has been enabled by some councils on most content types:

- initial page content to be entered in the body field
- further content such as contacts, documents and text can be added in paragraphs, either in the same tab as the body content or a different one

This issue is contentious as it takes us away from the simplicity of the original design. It's discussed fairly regularly though: at LGD Camp in 2024, and again in a recent [Content Group](https://miro.com/app/board/uXjVLtm7YLo=/?moveToWidget=3458764623902099166&cot=14)

There would be less of a problem if we [limit the paragraph types that can be used](https://github.com/localgovdrupal/localgov_paragraphs/issues/254)

Let's try this out with a service page and decide if it's an improvement

### This is done when...

- We've agreed how to page building elements to a Service page
- The fixes are listed here
- The fixes can be tested by selected users

**Comment by LizziCBFC** [comment]
Yes @willguv - we have it on pretty much all our main content types with the exception of news articles, form start pages, events, testimonials, and landing pages.

It does require more governance to make sure we only use the elements when appropriate, but provides a lot more flexibility with the content building process.

**Comment by finnlewis** [comment]
This has come up at Greenwich recently too, looking ahead to whether we want to add components and paragraphs (layout / images etc) to service landing pages / sub landing pages / pages. 

The original structure of the service content types was driven b yresearch from Clearleft at Brighton, and GCD guidelines. 

The outcome was service content types (landing / sublanding / page) with just text, no layout, no images, the idea being that most users find the information quicker if they are not distracted by calls to action, images and varying layouts. (I imagine this also requires a decent information architecture.)

But all that research and GDS guidance was a long time ago.

Can we co-ordinate and generate some updated user research or other evidence to help steer our 'best practice' guidance and what we advise as a community? 

How would we go about it? 

What are the questions we'd want to answer?

Does having layout options help users find content on service landing pages? 
Does having layout options help users consume content on a service page?
Does having images on service pages help users consume content?

**Comment by willguv** [comment]
@tonypaulbarker to make a demo using layout paragraphs

**Comment by markconroy** [comment]
- Issue: https://www.drupal.org/project/localgov_services/issues/3562044
- MR: https://git.drupalcode.org/project/localgov_services/-/merge_requests/7

Closing this issue. Let's continue the conversation over on Drupal.org.

**Comment by finnlewis** [comment]
Just discussing in our standup, and there's some confusion on the requirements here.

Reading back above: 

@LizziCBFC said they "have it on pretty much all our main content types"  - which I assumed meant layout paragraphs , but actually this might just be the option for multiple paragraph types in a single column. 

I was suggesting the need for layout paragraphs, to support multiple columns, in the same way that subsite pages work. This came from user needs at Greenwich. I also suggested some questions we might want to answer via user research to confirm if this is a route we want to take. 

Discussing today, @willguv thinks we don't want multiple column layouts on a service page, so we were just looking at simple paragraphs field with a few extra paragraph types. 

@markconroy suggests that even if we just want a single column with multiple paragraph type, we probably still want to use layout paragraphs, such that we have consistency in user experience, and templating / theming.

I think we concluded that we will run a charette in the new year with interested parties to align on the requirements and perhaps define acceptance tests.

**Comment by tonypaulbarker** [comment]
Moved to this Drupal.org issue -> https://www.drupal.org/project/localgov/issues/3567697
